src/Mod/Sketcher/Gui/SketcherToolDefaultWidget.h

```diff
@@ -87,25 +97,48 @@ class SketcherToolDefaultWidget : public QWidget
 
     void setParameterLabel(int parameterindex, const QString & string);
 
+    void setNoticeText(const QString& string);
+    void setNoticeVisible(bool visible);
+
+    void initNCheckbox(int ncheckbox);
+    void setCheckboxVisible(int checkboxindex, bool visible);
+    void setCheckboxChecked(int checkboxindex, bool checked);
+    void setCheckboxLabel(int checkboxindex, const QString& string);
+    bool getCheckboxChecked(int checkboxindex);
+    bool isCheckboxPref(int checkboxindex);
+    void setCheckboxPref(int checkboxindex, std::string prefEntry);
+    void setCheckboxPrefEntry(int checkboxindex, std::string prefEntry);
+
     template<typename F>
     boost::signals2::connection registerParameterValueChanged(F&& f) {
         return signalParameterValueChanged.connect(std::forward<F>(f));
     }
 
+    template<typename F2>

```

Comment:
There is no need to name the parameter differently "F2". 

No need to change it at this point.